hack/create-kind-cluster.sh

```diff
@@ -48,3 +62,21 @@ nodes:
   image: kindest/node:${NODE_VERSION}@${NODE_SHA}
 EOF
 
+# connect the registry to the cluster network if not already connected
+if [ "$(docker inspect -f='{{json .NetworkSettings.Networks.kind}}' "${REGISTRY_NAME}")" = 'null' ]; then
+  docker network connect "kind" "${REGISTRY_NAME}"
+fi
```

Comment:
Boy you must've spent a lot of time on this, thanks for that!

Reply:
I copy-pasted logic from the documentation https://kind.sigs.k8s.io/docs/user/local-registry/

Reply:
wanna give credit from there this is?